include/openmc/nuclide.h

```diff
@@ -127,6 +127,8 @@ class Nuclide {
   static int XS_FISSION;
   static int XS_NU_FISSION;
   static int XS_PHOTON_PROD;
+  static int XS_PROMPT_NU_FISSION;
```

Comment:
Pretty sure we should mark these as `static const int` since they are guaranteed to never change, and the compiler might optimize a little better if it knows that.

Reply:
I'm going to defer to the OpenMC team since I was just following precedent. I wanted to make XS_PROMPT_NU_FISSION and XS_DELAYED_NU_FISSION identical in form to XS_NU_FISSION. If there is a good case for modifying all these to static const int, let's do it in a separate pull request.

Reply:
Yeah, sounds good. The performance gain would be negligible anyways. I too dislike PR scope bloat, haha.